compiler/rustc_driver/src/pretty.rs

```diff
@@ -502,7 +502,8 @@ fn print_with_analysis(
                     thir::build_thir(tcx, ty::WithOptConstParam::unknown(did), &arena, &body.value);
                 let _ = writeln!(out, "{:?}:\n{:#?}\n", did, thir);
             }
-            out
+            out*/
+            todo!()
```

Comment:
seems like this file is still WIP ?

Reply:
I'd like not to block this on `-Zunpretty=thir-tree`, since landing this will unblock making THIR a query

Reply:
I should remove the code commented out though

Reply:
can we open an issue on the project-thir-unsafeck repo and reference it here in a comment? e.g., FIXME(rustlang/project-thir-unsafeck#123)